openwisp_ipam/api/views.py

```diff
@@ -137,10 +137,10 @@ def count(self):
         if self.stop is not None:
             return self.stop - self.start
         broadcast = int(self.subnet.subnet.broadcast_address)
-        # IPV4 (exclude broadcast)
         if self.subnet.subnet.max_prefixlen == 32:
```

Comment:
```suggestion
        if self.subnet.subnet.version == 4:
```

I was wrong with my initial findings. The code is using `max_prefixlen` to differentiate between IPv4 and IPv6 networks. Let's use the `version` attribute for this instead.